docs/Prelude.md

```diff
@@ -983,4 +983,20 @@ instance showArray :: (Show a) => Show (Array a)
 instance showOrdering :: Show Ordering
 ```
 
+#### `NaturalTransformation`
+
+``` purescript
+type NaturalTransformation f g = forall a. f a -> g a
+```
+
+A natural transformation
+
+#### `Natural`
+
+``` purescript
+type Natural f g = NaturalTransformation f g
```

Comment:
I think it'd be nice to pick one name and stick with it. I'd pick `NaturalTransformation` since the extra verbosity doesn't cost much and it's explicit.